packages/slice-machine/tests/project/cypress/integration/onboarding/00-redirects.spec.js

```diff
@@ -14,21 +14,18 @@ describe('onboarding redirects and local storage', () => {
     cy.saveLocalStorage();
   });
 
-  it('should redirect to /onboarding when is-onboared is not in local storage', () => {
+  it('should redirect to /onboarding when is-onboarded is not in local storage', () => {
     cy.intercept('POST', '/tracking/onboarding', ({body}) => {
       expect(body.id).equal("slicemachine_onboarding_start")
     })
     
     cy.visit('/')
 
     cy.location('pathname', {timeout: 1000}).should('eq', '/onboarding')
-
-    cy.wait(1500)
-    cy.getLocalStorage("is-onboarded").should('eq', 'true')
```

Comment:
why removing this ?
If you're onboarded it should be reflected on the local storage no ?

Reply:
As the external source of truth that makes sure you won't be onboarded again.

Reply:
because the feature changes, now the onbaording flag is set to true only when the user skip on finish the onboarding